iroha_schema/iroha_schema_bin/src/main.rs

```diff
@@ -16,13 +16,35 @@ macro_rules! to_json {
 }
 
 fn main() {
-    println!(
-        "{}",
-        to_json!(
-            VersionedTransaction,
-            VersionedSignedQueryRequest,
-            VersionedQueryResult,
-            VersionedEventSocketMessage,
-        )
-    )
+    // $ rg '^pub (struct|enum)' |cut -d' ' -f3|sed 's/[(< ].*//'
+    let json = to_json! {
+        Add,
+        And,
```

Comment:
Is this for python lib?

Reply:
Also with the top level structs that were previously here we did get all the ones that you listed here. No need to change this part as far as I can see.

Reply:
I guess we can miss some things. That is why shell one liner is there for regeneration.

Reply:
No, this is just for schema.

Reply:
Check my upper comment. Those 4 structs are exposed in our API, others that are needed for them are just recursively saved.

Reply:
We haven't forgotten anything - all our API is implemented in java client lib with current json :)
That's the point of our schema generator design, that we only outline the ones exposed in API.

Reply:
Added versioned types also. Kinda feel that it is better for that to be explicit.

Reply:
Why? I mean schema is designed only to expose API. And schema is needed for API :)
I do disagree here. We can also ask Rinat and Roman for their opinions. Maybe have a quick call right now.